src/components/View/Struttura/StrutturaContatti.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import { defineMessages, useIntl } from 'react-intl';
+import { CardSimple } from 'io-sanita-theme/components';
+import { RichTextSection } from 'io-sanita-theme/helpers';
+
+const messages = defineMessages({
+  contatti: {
+    id: 'contatti',
```

Comment:
```suggestion
    id: 'struttura_Contatti',
```
l'avevi corretto così nell'evento